docs/sdks/remove-api-key.md

```diff
@@ -31,11 +31,29 @@ from c8 import C8Client
 client = C8Client(protocol='https', host='gdn.paas.macrometa.io', port=443,
                         email='nemo@nautilus.com', password='xxxxx',
                         geofabric='_system')
+                        
+# Get a list of all the API key ID's names
+apiKeyIds = []
+for api in client.list_all_api_keys():
+    apiKeyIds.append(api["keyid"])
 
-# Remove api key
-remove = client.remove_api_key('id1')
+# Checks if the API key exists
+for apiElement in apiKeyIds:
+    if apiElement == CHOSEN_API_KEY_ID:
+        API_ACTIVE = True
```

Comment:
@jesusgonzalez12 I guess you forgot to declare CHOSEN_API_KEY_ID and API_ACTIVE on top after import C8Client.
Example:
API_ACTIVE = False
CHOSEN_API_KEY_ID = "YourApiKey"

Reply:
@jesusgonzalez12 The same is missing I guess in "docs/sdks/create-api-key.md" as well you can add the declaration of these two variables there too.

Reply:
Thanks @koshy-thomas. You're right, thanks for checking. Making changes.

PS: for this I have a single file that adds the key if the ID does not exists while removing it If the key already exists. So translating that to two separate files I had to change a few prints and picky-choose what I needed for each example. As I can see, I have room for improvement on that department.

Changes here are complete. Will now check PR 205.